guides/typography.md

```diff
@@ -1,146 +1,267 @@
+for the library's components. Additionally, Angular Material provides APIs for applying typography
+styles to elements in your own application.
 
+Angular Material's theming APIs are built with [Sass](https://sass-lang.com). This document assumes
+familiary with CSS and Sass basics, including variables, functions, and mixins.
 
-The typography levels are collected into a typography config which is used to generate the CSS.
+[theming-system]: https://material.angular.io/guide/theming
 
-## Usage
+## Including font assets
 
-To get started, you first include the `Roboto` font with the 300, 400 and 500 weights.
-You can host it yourself or include it from [Google Fonts][2]:
+By default, Angular Material is configured to use [Google's Roboto font][roboto] with the 300, 400,
+and 500 font-weight styles. To use Roboto, your application must load the font, which is
+not inlcuded with Angular Material. The easiest way to load Roboto is by using Google Fonts. The
+following snippet can be placed in your application's `<head>` to load Roboto from Google Fonts.
 
 ```html
 <link rel="preconnect" href="https://fonts.gstatic.com">
 <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
 ```
 
-Now you can add the appropriate CSS classes to the elements that you want to style:
+See [Getting Started with the Google Fonts API][fonts-api] for more about using Google Fonts.
 
-```html
-<h1 class="mat-display-1">Jackdaws love my big sphinx of quartz.</h1>
-<h2 class="mat-h2">The quick brown fox jumps over the lazy dog.</h2>
-```
+[roboto]: https://fonts.google.com/share?selection.family=Roboto:wght@300;400;500
+[fonts-api]: https://developers.google.com/fonts/docs/getting_started
 
-By default, Angular Material doesn't apply any global CSS. To apply the library's typographic styles
-more broadly, you can take advantage of the `mat-typography` CSS class. This class will style all
-descendant native elements.
+## Typography levels
 
-```html
-<!-- By default, Angular Material applies no global styles to native elements. -->
-<h1>This header is unstyled</h1>
+A **typography level** is a collection of typographic styles that corresponds to a specific
+part of an application's structure, such as a header. Each level includes styles for font family,
+font weight, font size, and letter-spacing. Angular Material uses the [typography levels
+from the 2014 version of the Material Design specification][2014-typography], outlined in the
+table below.
 
-<!-- Applying the mat-tyography class adds styles for native elements. -->
-<section class="mat-typography">
-  <h1>This header will be styled</h1>
-</section>
-```
+| Name            | Description                                                                 |
+|-----------------|-----------------------------------------------------------------------------|
+| `display-4`     | Large, one-off header, usually at the top of the page (e.g. a hero header). |
```

Comment:
TBH I don't really know what differentiates them.

Reply:
I meant semantically, since they're all just different flavors of big. I think you're right, though, that explicitly saying the font-size is really the only meaningful way to convey the difference.